python/pyspark/sql/functions.py

```diff
@@ -173,16 +172,26 @@ def _():
 
 _functions_2_1 = {
     # unary math functions
-    'degrees': 'Converts an angle measured in radians to an approximately equivalent angle ' +
-               'measured in degrees.',
-    'radians': 'Converts an angle measured in degrees to an approximately equivalent angle ' +
-               'measured in radians.',
+    'degrees': """Converts an angle measured in radians to an approximately equivalent angle
+               measured in degrees.
```

Comment:
Let's add a newline here. Seems doc could be broken.

Reply:
added